pom.xml

```diff
@@ -217,6 +217,15 @@
   </dependencyManagement>
 
   <build>
+    <resources>
```

Comment:
Do we really need this? Which value do we read from the POM?

Reply:
We use it to write the value of lgs.person.data.processor.version to the application.properties